ligo/stablecoin/actions.ligo

```diff
@@ -103,44 +103,44 @@ function ensure_is_paused
  * that balance given.
  *)
 function credit_to
-  ( const parameter : mint_param_
+  ( const parameter : mint_param
   ; const store     : storage
   ) : storage is block
 { var updated_balance : nat := 0n
-; case store.ledger[parameter.to_] of
-    Some (ledger_balance) -> updated_balance := ledger_balance
+; case store.ledger[parameter.to_] of [
+  | Some (ledger_balance) -> updated_balance := ledger_balance
   | None -> skip
-  end
+  ]
 ; updated_balance := updated_balance + parameter.amount
 ; const updated_ledger : ledger =
     Big_map.update(parameter.to_, Some (updated_balance), store.ledger)
 } with store with record [ ledger = updated_ledger ]
 
-type debit_param_ is record
```

Comment:
I can't remember why we had this "_" suffix for these types, but is this no longer required?

Reply:
Oh, I remember now. So these labels are not forbidden anymore, right?